advice_animal/cli.py

```diff
@@ -188,6 +188,20 @@ def check(ctx: click.Context, target: str) -> None:
             )
 
 
+@main.command()
+@click.pass_context
+def list(ctx: click.Context) -> None:
+    runner = Runner(Path(ctx.obj.advice_path), inplace=False, mode=Mode.check)
+    for advice_name, check_cls in runner.iter_check_classes(
+        FixConfidence.UNSET, True, re.compile(".*")
+    ):
+        if check_cls.confidence.name != "UNSET":
+            name = click.style(advice_name, fg=check_cls.confidence.name.lower())
+        else:
+            name = click.style(advice_name, fg="green")
```

Comment:
nit: UNSET is the most restrictive, I'd probably go for something "worse" than red, like magenta.

Reply:
Ah. I assumed UNSET would default to green. My bad.